tests/rules/javascript/test.event_listener_fourth.js

```diff
@@ -0,0 +1,128 @@
+import JavaScriptScanner from 'scanners/javascript';
+import { VALIDATION_WARNING } from 'const';
+import { singleLineString } from 'utils';
+import * as messages from 'messages';
+
+describe('event_listener_fourth', () => {
+  it('should not allow a true literal', () => {
+    var code = 'window.addEventListener("click", function(){}, false, true);';
```

Comment:
Yep the nicest thing I think we can do here is keep the many rules but encapsulate more functionality into the utils helpers. That way we can keep things dry but also retain as much of the eslint way of doing things as possible.